chat-plugins/mafia.js

```diff
@@ -1029,6 +1037,28 @@ exports.commands = {
 			game.updateHost();
 		},
 
+		subhost: function (target, room, user) {
+			if (!this.canTalk()) return;
+			if (!target) return this.parse('/help mafia subhost');
+			this.splitTarget(target);
+			let targetUser = this.targetUser;
+			if (!targetUser || !targetUser.connected) return this.errorReply(`The user "${this.targetUsername}" was not found.`);
+			if (!room.users[targetUser.userid]) return this.errorReply(`${targetUser.name} is not in this room, and cannot be hosted.`);
+			if (!user.can('mute', null, room)) return this.errorReply(`/mafia subhost - Access denied.`);
```

Comment:
just use `this.can` since theres no reason for a custom error message.

Also you should probably check permissions earlier.